unittests/test_using_virtual_board/test_from_file_connector.py

```diff
@@ -90,16 +90,6 @@ def test_simple(self):
         ]
         self.check_other_connect(as_list)
 
-    def test_list_too_big(self):
```

Comment:
Why remove this test?

If it no longer works check the pop

Reply:
Restored - must have removed thinking it couldn't work in unit test, but actually found issues!